packages/flutter/lib/src/gestures/tap_and_drag.dart

```diff
@@ -400,12 +403,28 @@ class TapDragEndDetails with Diagnosticable {
   /// the number in the series this tap is.
   final int consecutiveTapCount;
 
+  /// The pointer's global position when it triggered this update.
```

Comment:
nit: maybe this should say `The global position at which the pointer lifted from the screen.` similar to `LongPressEndDetails`.

Reply:
Oh these are just copy-paste. Should I update them or keep them as previous?

Reply:
I'm leaning towards the updated version being more descriptive.